File: src/main/java/com/yanggang/refactoring/libraryapp/dto/book/request/BookRequest.java
package com.yanggang.refactoring.libraryapp.dto.book.request;

import lombok.Getter;
import lombok.NoArgsConstructor;

@Getter
@NoArgsConstructor
public class BookRequest {

    private String name;

    public BookRequest(String name) {
        this.name = name;
    }
}

File: src/main/kotlin/com/yanggang/refactoring/libraryapp/dto/book/request/BookRequest.kt
package com.yanggang.refactoring.libraryapp.dto.book.request;

import lombok.NoArgsConstructor;

@NoArgsConstructor
public class BookRequest {

    private String name;

    public BookRequest(String name) {
        this.name = name;
    }

    public String getName() {
        return name;
    }
}
